lib/cfg-parser.c

```diff
@@ -86,6 +86,7 @@ static CfgLexerKeyword main_keywords[] =
   { "stats_lifetime",     KW_STATS_LIFETIME },
   { "stats_level",        KW_STATS_LEVEL },
   { "stats",              KW_STATS_FREQ, KWS_OBSOLETE, "stats_freq" },
+  { "stats_max_dynamics", KW_STATS_MAX_DYNAMIC },
```

Comment:
Maybe it will be "punctual", sorry about that. 
I think "stats_max_dynamics" should be only "stats_max_dynamic". I know it is only just a "name" here.